packages/router/src/router.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/router
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/**
+ * Interface for router implementation
+ */
+export interface Router<T, R, K> {
```

Comment:
Let's use more descriptive names for these template args please, e.g.

```suggestion
export interface Router<RouteEntry, ResolvedRoute, Request> {
```